src/Resources/assets/controllers/accordion_controller.js

```diff
@@ -1,70 +1,55 @@
 import { Controller } from '@hotwired/stimulus';
-import * as StickyThead from 'stickythead'
 
 export default class extends Controller {
-    static targets = ['content', 'arrow']
+    static targets = ['header', 'content', 'arrow']
+
+    toggle(event)
+    {
+        /** @type {HTMLElement[]} */
+        const contentTargets = this.contentTargets;
+
+        /** @type {HTMLElement[]} */
+        const headerTargets = this.headerTargets;
+
+        /** @type {HTMLElement[]} */
+        const arrowTargets = this.arrowTargets;
 
-    toggle(event) {
         const current = event.currentTarget;
-        const arrow = current.querySelector('[data-araise--table-bundle--accordion-target=arrow]');
-        const isOpen = current.dataset.ariaExpanded == 'true';
-        const nextSiblings = this.nextUntil(current, '[data-action="click->araise--table-bundle--accordion#toggle"]');
+        const header = this.closestTarget(current, 'header');
+        const arrow = this.closestTarget(current, 'arrow');
+        const content = header.nextElementSibling;
+
+        const isOpen = header.getAttribute('aria-expanded') === 'true';
 
-        if(isOpen) {
+        // Reset all accordions
+        headerTargets.forEach((header) => {
+            header.setAttribute('aria-expanded', 'false');
+        });
+        arrowTargets.forEach((arrow) => {
             arrow.classList.remove('rotate-90');
-            current.dataset.ariaExpanded = 'false';
+        });
+        contentTargets.forEach((content) => {
+            content.classList.add('hidden');
```

Comment:
All those classes are coming from Tailwind; therefore these have to be submitted to the controller by the [Stimulus Classes API; otherwise they are statically bound to Tailwind. That's not what we want with Stimulus controllers.

So there should be an input possibility for adding expanded/collapsed classes.

(or they are selected via the aria-expanded and `:has`, with whatever API Tailwind has to add child selectors)